k8s/production/prometheus/release.yaml

```diff
@@ -139,6 +139,8 @@ spec:
       metricLabelsAllowlist:
         - pods=[*]
         - nodes=[*]
+      metricAnnotationsAllowList:
+        - pods=[gitlab/ci_job_id,metrics*]
```

Comment:
Is there any specific reason `gitlab/ci_job_id` is exposed as the only "gitlab" annotation?

Reply:
I did that so I could query the annotation by the job ID, and the other gitlab variables are more about the runner itself rather than the job imo